### PR TITLE
Add Bicep Docs

### PR DESCRIPTION
**Pull request by tjoudeh**
Minor updates on bicep modules

### CODE REVIEW

**Comment by pankajagrawal16** [comment]
I believe we can improve the UX a bit and make it consistent with rest of modules now. But may be we can do this after the content itself is in place and merged. So that we dont too doo many changes in this PR. I can look into updating the UX once this gets meged.

File: docs/aca/10-aca-iac-bicep/index.md [hunk 1]
```diff
@@ -2,7 +2,208 @@
...
+!!! note
+    To simplify the execution of the module, we will assume that the azure resource "Azure Container Registry" is already provisioned and it contains the latest images of the 3 services. We will not provision Azure Container Registry part of this Bicep modules.
+
+#### 1. Add the needed extension to VS Code
+You need to install an extension named [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep), This extension will simplify building Bicep files as they will offer IntelliSense, Validation, listing all available resource types, etc..
+
+#### 2. Define an Azure Container Apps Environment
+Add a new folder named `Bicep` on the root project directory, then add another folder named `Modules`. Add a new file named `container-apps-environment.bicep`. The content of the file can be found on this [link.](https://github.com/Azure/aca-dotnet-workshop/blob/main/bicep/modules/container-apps-environment.bicep).
+
+What we've added to this file is the following:
```
Comment (pankajagrawal16) [inline docs/aca/10-aca-iac-bicep/index.md:41]: Bullet list is bit off in the preview url https://azure.github.io/aca-dotnet-workshop/pr-preview/pr-54/aca/10-aca-iac-bicep/#2-define-an-azure-container-apps-environment, maybe it need an extra <kbd>Enter</kbd>

Comment (tjoudeh) [reply docs/aca/10-aca-iac-bicep/index.md:41]: Done.

**Review by waelkdouh — APPROVED** [review]
LGTM

**Review by waelkdouh — APPROVED** [review]
Completed Bicep Review. Ready to merge to the main branch.

**Pull request merged**